### PR TITLE
[Docs] Plugins technical review checklist

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

Since the very beginning of the Sylius plugins system, we are giving the availability to make your plugin officially [approved](https://docs.sylius.com/en/latest/book/plugins/approved-plugins.html). This technical review checklist is a detailed instruction what you should focus on, to make the approval process smoother and faster 🎉

### CODE REVIEW

File: docs/book/plugins/approved-plugins.rst [hunk 1]
```diff
@@ -20,11 +20,69 @@ Since Sylius is an open-source platform, there is a certain flow in order for th
...
 and general feeling. They will also ask you to provide some changes in the code (if needed) to make this plugin approved.
 
 **5.** Wait for your Plugin to be featured in `the list of plugins <http://sylius.com/developers/store/plugins>`_ on the Sylius website.
+
+.. _book_plugins_technical_requirements:
+
+Technical requirements
+----------------------
+
+Below you can find a list of requirements that your plugin needs to fulfill to be approved by Sylius core-team. Try to follow
```
Comment (pamil) [inline docs/book/plugins/approved-plugins.rst]:
```suggestion
Below you can find a list of requirements that your plugin needs to fulfill to be approved by Sylius Core Team. Try to follow
```

File: docs/book/plugins/approved-plugins.rst [hunk 2]
```diff
@@ -20,11 +20,69 @@ Since Sylius is an open-source platform, there is a certain flow in order for th
...
+**Coding standards:**
+
+* Does the code apply at least `PSR-1 <https://www.php-fig.org/psr/psr-1/>`_?
+
+Could have
+##########
+
+**Coding standards:**
+
+* Are there any apparent violations of one or more SOLID principles?
```
Comment (pamil) [inline docs/book/plugins/approved-plugins.rst]: If it's a checklist, answering with "yes" should mean I'm meeting the criteria.

```suggestion
* Does the code follow SOLID principles?
```

File: docs/book/plugins/approved-plugins.rst [hunk 3]
```diff
@@ -20,11 +20,69 @@ Since Sylius is an open-source platform, there is a certain flow in order for th
...
+**Installation:**
+
+* Is it possible to install the plugin on a fresh Sylius-Standard application with no problems?
+* Is every step needed for installation and configuration explained in the documentation? Are there any assumptions that could be confusing for less experienced developers?
+
+**Tests:**
+
+* Are there any unit tests for the plugin's classes? They can be written in PHPSpec, PHPUnit or any other working and reliable unit testing library
+* Does the unit tests cover at least the most crucial classes in the plugin (those which contain important business logic)?
+* Does the plugin contain some functional/acceptation tests (written in Behat/PHPUnit or similar tool)?
```
Comment (GSadee) [inline docs/book/plugins/approved-plugins.rst]:
```suggestion
* Does the plugin contain some functional/acceptance tests (written in Behat/PHPUnit or similar tool)?
```

**Comment by GSadee** [comment]
Thank you, Mateusz! :1st_place_medal:

**Pull request merged**